docs-src/concepts/what-is-a-list-filter.md

```diff
@@ -31,19 +31,17 @@ A List Filter contains [Search Attribute](/concepts/what-is-a-search-attribute)
   - **BETWEEN ... AND**
   - **IN**
   - **ORDER BY**
-  - **LIKE**
-
-- Wildcards ('\*', '%', etc.) are supported for string-type List Filter Search Attributes.
-  Use wildcards with the **LIKE** operator to query possible values:
-
-  ```bash
-  // Create a "ProductId" custom Search Attribute of type String
-  tctl admin cluster add-search-attributes --name ProductId --type String
-  // Match values that start with "book"
-  ProductId LIKE "book%"
-  // Match values that contain "favorite"
-  ProductId LIKE "%favorite%"
-  ```
+
+  The last operator (**ORDER BY**) is supported only with Elasticsearch as the Visibility store.
+  Custom Search Attributes of `Text` type cannot be used in **ORDER BY** clauses.
+
+- To use a partial string match (i.e. specific word), create a [custom Search Attribute](/app-dev-context/observability#custom-search-attributes) of type `Text` and set it value to a phrase that contains that word.
```

Comment:
I would double check this with @alexshtin if this is by default whole-word search or partial as in https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-regexp-query.html#_standard_operators is allowed too.

Reply:
ah there is sample below. okok sry